src/apps/transfer_ownership/api.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+
+from fastapi import Body, Depends
+
+from apps.authentication.deps import get_current_user
+from apps.transfer_ownership.domain import InitiateTransfer, TransferResponse
+from apps.transfer_ownership.service import TransferService
+from apps.users.domain import User
+
+__all__ = [
+    "transfer_initiate",
+    "transfer_respond",
+]
+
+
+async def transfer_initiate(
+    applet_id: int,
+    user: User = Depends(get_current_user),
+    transfer: InitiateTransfer = Body(...),
+) -> None:
+    """Initiate a transfer of ownership of an applet."""
+    await TransferService(user).initiate_transfer(applet_id, transfer)
+
+
+async def transfer_respond(
+    applet_id: int,
+    key: uuid.UUID,
+    user: User = Depends(get_current_user),
+    response: TransferResponse = Body(...),
+) -> None:
+    """Respond to a transfer of ownership of an applet."""
+    await TransferService(user).respond_transfer(applet_id, key, response)
```

Comment:
Don't you think that on transferring the ownership we have to return the new instance of the applet?

Reply:
Why do we need the `TransferResponse` there?
It has only one boolean field inside. Regarding the REST, we have to create 2 different endpoints: `HTTP POST /{applet_id}/transferOwnership/{key}` to allow that or `HTTP DELETE /{applet_id}/transferOwnership/{key}` to reject the transfer.
What do you think?

Reply:
In that case, we also can exclude the TransferResponse data model from our project. Also, it is a little bit weird in my opinion that we have a response field that is a request argument, don't you think?
It might confuse new developers in the future.